scripts/sct_deepseg_sc.py

```diff
@@ -410,6 +416,68 @@ def heatmap2optic(fname_heatmap, lambda_value, fname_out, z_max, algo='dpdt'):
         ctr_nii.save()
 
 
+def find_centerline(algo, image_fname, path_sct, contrast_type, brain_bool, folder_output, remove_temp_files):
+
+    if algo == 'svm' and Image(image_fname).dim[2] > 1:  # isct_spine_detect requires nz > 1
```

Comment:
why not checking beforehand if the image is 2D, and if it is, duplicate the slice to create a 3D volume, then run the centerline detection, then crop back to 2D on the output centerline.
This will prevent from having an error with the default parameters if 2D image is input.

Reply:
Excellent idea, thank you.
It is ready to be reviewed (commit #5d2f492).